custom_components/healthchecksio/config_flow.py

```
@@ -135,41 +132,49 @@ async def async_step_user(
                     )
                 self._errors["base"] = "auth"
 
-        DATA_SCHEMA: vol.Schema = vol.Schema({
-            vol.Required(
-                CONF_API_KEY,
-                default=user_input.get(CONF_API_KEY) if user_input is not None else None,
-            ): str,
-        })
+        DATA_SCHEMA: vol.Schema = vol.Schema(
```

Pylint (3.3.7)

[convention] Variable name "DATA_SCHEMA" doesn't conform to snake_case naming style

(C0103)